{"name": "\"We can't find your account\" message is displayed when user enters incorrect credentials", "status": "passed", "steps": [{"name": "Given Open Target.com", "status": "passed", "start": 1727726442490, "stop": 1727726442983}, {"name": "When User presses Sign In button", "status": "passed", "start": 1727726442983, "stop": 1727726445365}, {"name": "And User presses Sing In button on expanding panel", "status": "passed", "start": 1727726445365, "stop": 1727726445994}, {"name": "Then Sign in form page is displayed", "status": "passed", "start": 1727726445994, "stop": 1727726446416}, {"name": "When user enters incorrect email address", "status": "passed", "start": 1727726446417, "stop": 1727726446517}, {"name": "And user enters incorrect password", "status": "passed", "start": 1727726446517, "stop": 1727726446586}, {"name": "And User presses Sing In button in Sign In form", "status": "passed", "start": 1727726446586, "stop": 1727726446640}, {"name": "Then verify 'We can't find your account' message is shown", "status": "passed", "start": 1727726446640, "stop": 1727726446888}], "start": 1727726434459, "stop": 1727726449026, "uuid": "f4ac98d9-ce4b-44c2-87a3-72a738738968", "historyId": "5faf66ffaa00fb5b73613c989806cc3e", "fullName": "Tests for sign in functionality: \"We can't find your account\" message is displayed when user enters incorrect credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Tests for sign in functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}